core/dbt/parser/schemas.py

```diff
@@ -314,7 +316,7 @@ def _parse_generic_test(
         # is not necessarily this package's name
         fqn = self.get_fqn(fqn_path, builder.fqn_name)
 
-        # this is the config that is used in render_update
+        # this is the ContextConfig that is used in render_update
```

Comment:
Is it possible to use mypy to make this assertion here?

Reply:
Sure. Have changed the following line to say config: ContextConfig